src/app/portlets/content-types/fields/content-type-fields-variables/content-type-fields-variables.component.ts

```diff
@@ -87,35 +87,14 @@ export class ContentTypeFieldsVariablesComponent implements OnInit, OnChanges, O
     /**
```

Comment:
We need to prefix all things in the app with `dot-` (for files and folders) and `Dot` for class names